docs/pages/getting-started/4-connecting-crm.rst

```diff
@@ -0,0 +1,216 @@
+Requests that are made in the context of the organization will use the private-key of the organization.
+To easily control the DID Document of an organization, the vendor will be the controller.
+
+Create and store a customer DID
+===============================
+
+A DID can be created like the vendor DID:
+
+.. code-block:: text
+
+    POST <internal-node-address>/internal/vdr/v1/did
+    {
+        "selfControl": false,
+        "controllers": [{did}],
+        "assertionMethod": true,
+        "capabilityInvocation": false
+    }
+
+Where ``{did}`` must be replaced with the vendor DID.
+The body for creating an organization DID differs from the vendor DID in the fact that the vendor DID is in control of the newly generated DID Document.
+The ``assertionMethod`` is still true since it'll allow for the generation of access-tokens in the context of the organization.
+The result is similar to the output of the vendor DID creation.
+In this case the ``id`` must also be extracted and stored within the vendor CRM for the right organization.
+
+Issue a Nuts Organization Credential
+====================================
+
+After registering an organization, its presence on the network and in the Nuts registry is now only a DID.
+In order for other organizations to find the correct DID and connected services, credentials should be issued and published over the network.
+For this, the *NutsOrganizationCredential* can be issued by any vendor.
+A *NutsOrganizationCredential* contains the ``name`` of the organization and the ``city`` where this name is registered as organization.
+The combination of those should be unique (since duplicate names within a sector is disallowed).
+
+A credential can be issued with the following call:
+
+.. code-block:: text
+
+    POST <internal-node-address>/internal/vcr/v1/vc
+    {
+        "type": "NutsOrganizationCredential",
+        "issuer": "{did}",
+        "credentialSubject": {
+            "name": "{name}",
+            "city": "{city}"
+        }
+    }
+
+Where ``{did}`` must be replaced with the vendor DID. ``{name}`` and ``{city}`` must be replaced with the correct information.
+The API will respond with the full Verifiable Credential. It's not required to do anything with that since issued credentials can be found again.
+:ref:`This page <vc-concepts>` contains some more information on specific credentials.
+
+Trusting other vendors as issuer
+================================
+
+A node operator must not blindly trust all the data is published over the network. Before credentials can be found, the issuer has to be trusted.
+By default, no issuers are trusted. A list of untrusted issuers can be obtained from the node through:
+
+.. code-block:: text
+
+    GET <internal-node-address>/internal/vcr/v1/NutsOrganizationCredential/untrusted
+
+This will return a list of all DIDs that are currently not trusted. If a DID is to be trusted should be validated out-of-band.
```

Comment:
it should be validated out-of-band (missing "it")

Does "everyone" know what out-of-band is or do we need to provide an example?

Reply:
"it" isn't needed. I added an example.